### PR TITLE
Add GitHub files, copyright and License information

### PR DESCRIPTION
**Pull request by gwarf**
Add GitHub files, copyright and License information to be used as templates for other projects.

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Code of Conduct
+
+This code of conduct applies to the maintainers and contributors alike.
+
+## Dealing with issues and support requests
+
+_We wish to add a specific section on dealing with issues opened against the
+repository here._
+
+This repository exists in the context of EGI Operations. While that scope does
```
Comment (enolfc) [inline .github/CODE_OF_CONDUCT.md]: EGI Foundation?

File: .github/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+not restrict the usage, it does inform the priority we assign to issues and the
+order we deal with them.
+
+We welcome issues reported by the public, and more specifically the community
+of people using this repository.
+The EGI operations team is small and cannot support all requests equally.
```
Comment (enolfc) [inline .github/CODE_OF_CONDUCT.md]: Foundation?

File: .github/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+While we undertake to do everything in our power to respond to issues
+timeously, and to prioritise issues based on reasonable requests from
```
Comment (enolfc) [inline .github/CODE_OF_CONDUCT.md]: timely? timeously seems Scottish

File: .github/CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an
+appointed representative at an online or offline event. Representation of a
+project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at contact@egi.eu. The project team
```
Comment (enolfc) [inline .github/CODE_OF_CONDUCT.md]: I would not call contact@egi.eu the "project team", maybe project -> EGI

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+be easier to send pull requests.
+
+### Branches and Commits
+
+You should submit your patch as a git branch named after the Github issue, such
+as `#3`\. This is called a _topic branch_ and allows users to associate a
+branch of code with the ticket.
+
+It is a best practice to have your commit message have a _summary line_ that
+includes the ticket number, followed by an empty line and then a brief
```
Comment (enolfc) [inline .github/CONTRIBUTING.md]:
```suggestion
includes the issue number, followed by an empty line and then a brief
```

File: AUTHORS [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Maintainers
```
Comment (enolfc) [inline AUTHORS:1]: Will this file appear on every repo that we create?

File: COPYRIGHT [hunk 7]
```diff
@@ -0,0 +1,4 @@
+This project is licensed under MIT.
```
Comment (enolfc) [inline COPYRIGHT:1]: I was wondering last days whether APL was the right one or not  for us. Honestly I can't say :(

**Review by enolfc — CHANGES_REQUESTED** [review]
Thanks @gwarf.

Check some comments I raised.

File: COPYRIGHT (hunk 7 above)
Comment (gwarf) [reply COPYRIGHT:1]: I find MIT much more straightforward, don't like all the APL stuff. In the end they are both permissive licenses, and I don't like to endorse their patent-related stuff

Comment (enolfc) [reply COPYRIGHT:1]: +1 MIT then

File: AUTHORS (hunk 6 above)
Comment (gwarf) [reply AUTHORS:1]: yes, but it's a a tempalte, things can be removed/adapted as needed

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (gwarf) [reply .github/CODE_OF_CONDUCT.md]: EGI Federation

File: .github/CODE_OF_CONDUCT.md (hunk 2 above)
Comment (gwarf) [reply .github/CODE_OF_CONDUCT.md]: EGI team

File: .github/CODE_OF_CONDUCT.md (hunk 3 above)
Comment (gwarf) [reply .github/CODE_OF_CONDUCT.md]: OK :)

File: .github/CODE_OF_CONDUCT.md (hunk 4 above)
Comment (gwarf) [reply .github/CODE_OF_CONDUCT.md]: OK.

File: AUTHORS (hunk 6 above)
Comment (gwarf) [reply AUTHORS:1]: I've replaced by a placeholder/sample.

File: COPYRIGHT (hunk 7 above)
Comment (gwarf) [reply COPYRIGHT:1]: I don't want to support the idea that software patent can be legit, especially here in Europe...

**Review by enolfc — APPROVED** [review]
👍

**Pull request merged**